Sources/UIView+Sweet.swift

```diff
@@ -66,9 +66,12 @@ public extension UIView {
 
     /// Helper method to instantiate a nib file.
     ///
+    /// - Parameter nibName: The name of the nib to be loaded. If none is passed, it uses the class name. Defaults to nil.
     /// - Parameter bundle: The bundle where the nib is located, by default we'll use the main bundle.
+    ///
     /// - Returns: Returns an instance of the nib as a UIView.
-    class func instanceFromNib<T: UIView>(bundle: Bundle = .main) -> T {
-        return UINib(nibName: String(describing: T.self), bundle: bundle).instantiate(withOwner: nil, options: nil)[0] as! T
+    public class func instanceFromNib<T: UIView>(nibName: String? = nil, bundle: Bundle = .main) -> T? {
+        let name = nibName ?? String(describing: T.self)
+        return UINib(nibName: name, bundle: bundle).instantiate(withOwner: nil, options: nil)[0] as? T
```

Comment:
do we always want to have `owner` `nil` ? 🤔

Reply:
We could probably add owner and options to the fold. 👍